src/bitfield.cpp

```diff
@@ -41,6 +41,8 @@ namespace libtorrent
 {
 	bool bitfield::all_set() const
 	{
+		if(m_buf == nullptr) return false;
```

Comment:
would it make more sense to say: ``if (size() == 0) return false;`` ?
it feels like that would be more intuitive and direct